### ISSUE TITLE
Flaky unit tests with account_cache_test.go

### ISSUE THREAD

**Issue by mrsmkl** [celo-org/celo-blockchain#629]
### Expected Behavior

Test success.

### Current Behavior

```
--- FAIL: TestWatchNoDir (12.70s)
    account_cache_test.go:132: 
        got  []
        want [{[126 245 166 19 95 31 214 160 37 147 238 220 134 156 109 65 217 52 174 248] keystore:///tmp/eth-keystore-watch-test-7167-8582289867329813576/aaa}]
FAIL
```

https://circleci.com/gh/celo-org/celo-blockchain/16237

**Comment by oneeman** [comment]
This doesn't reproduce locally and has not been mentioned in the go-ethereum github issues, it only happens on CircleCI, suggesting it's due to the machines being slower and overloaded.  As a first step, I opened a PR to bump up the CI resource class to make the machines more powerful.  If that's still not enough, we can increase further, increase the test's delay (it current waits a maximum of about 8 seconds), or make this test not run with `t.Parallel()` so there is less load on the machine when it runs.

**Comment by oneeman** [comment]
Closed by #1269, will reopen if it still happens after that.
